### DISCUSSION TITLE
Alert Maintenance

### DISCUSSION THREAD

**Discussion by pkharris** [datalust/seq-tickets#1530] [Help] [answered]
Seq Version: 2021.4
User Role: Admin

- Viewing Settings/Notifications I can see other user's alerts.  I am only able to disable these.

- Viewing Alerts from the Alerts tab, I am only able to see my alerts.  I would like to be able to see, edit, and delete any alert (everyone elses's).

Please advise.

**Comment by nblumhardt** [comment, marked as answer]
Thanks for the note! There's no way to do this today, but it's useful feedback for us to consider. In the medium- to long-term we're planning to separate control of visibility from ownership, where I think this would be relevant.

Today, it'll be necessary for other users to mark their alerts "shared" in order for admins to edit them.

HTH,
Nick

**Reply by pkharris** [reply]
Thank you.  This will work and I have informed my team.
